index.html

```diff
@@ -15,176 +15,247 @@
-                    required. has been the industry's standard dummy text ever since the 1500s, when an unknown
-                    printer took a standard dummy text.</p>
-                <ul class="first-buttons">
-                    <li class="list">css</li>
-                    <li class="list">html</li>
-                    <li class="list1">bootstrap</li>
-                    <li class="list">Ruby</li>
-                </ul>
-                <button type="button" class="button1">See Project</button>
+        <div class="info">
+             <div class="icon">
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-angellist"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
             </div>
-        </section>
-        <section class="Cards">
-            <div class="arts">
-                <div class="btn-list">
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard</p>
-                        <ul>
-                            <li class="list-btn" >html</li>
-                            <li class="list-boot">bootstrap</li>
-                            <li class="list-btn">Ruby</li>
-                        </ul>
-                    <button type="button" class="btn1">See Project</button>
+            <div class="about-bruk">
+                <h1 class="bura">Hey There. I'm Bruk </h1>
+                <h2 class="job">I'm Software Developer</h2>
+                <p class="discription"> I can help you build a product , feature or website Look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+        </div>
+    <main class="work-pro">
+        <div class="body">
+            <div class="recent">
+                <div class="work-hr">
+                    <h2 class="my-recent">My Recent Works</h2> 
+                    <hr>
                 </div>
-                <div class="btn-list">
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard</p>
-                        <ul>
-                            <li class="list-btn">html</li>
-                            <li class="list-boot">bootstrap</li>
-                            <li class="list-btn">Ruby</li>
+                <div class="img">
+                    <img src="img/placeholder-new.png" alt="">
+                    <div class="my-workr">
```

Comment:
- [ ] If you see the Figma design then the image takes 60% and content takes 40% So kindly Implement it as Figma.
### your design 
![his](https://user-images.githubusercontent.com/81727988/166093617-a96f201c-496e-4dc0-b8a2-6c64e7fb3ef4.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/166093621-5a29398f-b1f1-43ca-8742-3f1a43af5cfd.png)